chart/Chart.yaml

```diff
@@ -4,7 +4,7 @@ name: fleet
 keywords:
   - fleet
   - osquery
-version: 3.9.0
+version: 3.9.1
```

Comment:
Is it common for Helm charts to have versions different from that of the software they are deploying? Should we consider making this `3.9.0-1` or something like that?

cc @jalseth

Reply:
I've seen both, but I have a strong preference to tying the chart version to the app version. It removes a lot of confusion. I recommend leaving the version as-is (`3.9.0`) and when the next release is made, this version will be bumped and a new chart will be released. 

I would only consider releasing a new version of the Helm chart outside of the normal release cycle if there is a bug in the chart itself. And then a `-1` etc suffix would make sense.

Reply:
I'm inclined to agree with @jalseth that we will only bump the chart version as part of the release process (as documented in https://github.com/fleetdm/fleet/blob/master/docs/3-Contribution/5-Releasing-Fleet.md).

Reply:
i keept `3.9.0`